components/cache/cache_invalidation.rst

```diff
@@ -0,0 +1,104 @@
+.. index::
+   single: Cache Invalidation
+   single: Cache Tags
+   single: Cache Namespaces
+
+Cache Invalidation
+==================
+
+Cache invalidation is the process of removing all cached items related to a
+change in the state of your model. The most basic kind of invalidation is direct
+items deletion. But when the state of a primary resource has spread accross
+several cached items, keeping them in sync can be difficult.
+
+The Symfony Cache component provides three mechanisms to help solve this problem:
+
+- tags based invalidation for managing data dependencies;
+- namespace based invalidation for context and sub-contexts dependend data;
+- expiration based invalidation for time related dependencies.
+
+.. versionadded:: 3.2
+    Tags and namespace based invalidation were introduced in Symfony 3.2.
```

Comment:
I believe it should be "was" here, as it's refering to "X based invalidation", which is singular.